app/admin/forward_freight.rb

```diff
@@ -0,0 +1,50 @@
+ActiveAdmin.register ForwardFreight do
+
+# See permitted parameters documentation:
+# https://github.com/activeadmin/activeadmin/blob/master/docs/2-resource-customization.md#setting-up-strong-parameters
+#
+# permit_params :list, :of, :attributes, :on, :model
+#
+# or
+#
+# permit_params do
+#   permitted = [:permitted, :attributes]
+#   permitted << :other if params[:action] == 'create' && current_user.admin?
+#   permitted
+# end
+  permit_params :email, :password, :password_confirmation
```

Comment:
You're almost finished! :) Let's just set the strong parameters and the form fields like you did in business user and broker :)